superflore/generators/bitbake/ros_meta.py

```diff
@@ -74,17 +83,28 @@ def get_file_revision_logs(self, *file_path):
     def get_change_summary(self, distro):
         self.repo.git.add('meta-ros{0}-{1}/generated-recipes'.format(
             yoctoRecipe._get_ros_version(distro), distro))
+        self.repo.git.add('meta-ros{0}-{1}/conf/ros-distro/include/{1}/'
+                          'generated/*.inc'.format(
+                              yoctoRecipe._get_ros_version(distro), distro))
+        self.repo.git.add('meta-ros{0}-{1}/files/{1}/generated/'
+                          'rosdep-resolve.yaml'.format(
+                              yoctoRecipe._get_ros_version(distro), distro))
+        self.repo.git.add('meta-ros{0}-{1}/files/{1}/generated/'
+                          'newer-platform-components.list'.format(
+                              yoctoRecipe._get_ros_version(distro), distro))
         sep = '-' * 5
         return '\n'.join([
             sep,
             self.repo.git.status('--porcelain'),
             sep,
             self.repo.git.diff(
+                'HEAD',
```

Comment:
This is needed because of the newly added `git add`-s, right?

Reply:
yes